packages/fast-components-react-msft/src/action-toggle/README.md

```diff
@@ -0,0 +1,13 @@
+# Action Toggle
+*Action toggle* should be used as control on or around an interactive region of the page that affects the region or items within the region. It usually represents a switch between one of two states.
+
+## Usage
+Use *action toggle* for specific control around a functional state. Labeling can be done using a glyph and text, but could also contain just a glyph or just text. Make sure the text and glyph clearly represent the expected function. When only a glyph is used to label the button, include a tooltip in a span element following the button element.
```

Comment:
"When only a glyph is used to label the button, include a tooltip in a span element following the button element."
I'm not sure we should include this right now since tooltip doesn't exist.

Reply:
Tooltip is coming fairly soon. If we remove this we should file a task to re-add it once tooltip is completed.

Reply:
Seems like unnecessary churn to me to remove tooltip references only to restore them in the short term?

Reply:
I like keeping tooltip so long as we reference it in terms of "a tooltip" (generic pattern), rather than "tooltip" (component)

Reply:
This? -"When only a glyph is used to label the button other elements, such as a tooltip, may be required to make the toggle's function clear."  I think the previous blurb was overly prescriptive.